master/internal/config/provconfig/gcp_config_test.go

```diff
@@ -37,6 +37,9 @@ func TestUnmarshalGCPClusterConfig(t *testing.T) {
 	"zone": "test-zone",
 	"boot_disk_size": 100,
 	"boot_disk_source_image": "test-source_image",
+	"labels": {
+	    "test-labels-key": "test-labels-value"
```

Comment:
maybe have the strings here which are different from `label_key` / `label_value` so this could be a more natural test for "merge" behavior?

Reply:
The strings are slightly different ("labels" vs. "label"), which I admit is confusing.